### DISCUSSION TITLE
Only display and window decorations appear, nothing else

### DISCUSSION THREAD

**Discussion by randompersononinternet69** [PCBox/PCBox#82] [Q&A] [answered]
I wanted to try this project out for the first time, seemed pretty cool.
But well, for whatever reason it appears only the display is shown?
I don't know if I downloaded the wrong binary (Linux x64, Debian 12)
2 screenshots are attached
![image](https://github.com/PCBox/PCBox/assets/107446530/bd0cd957-827b-4f33-9f4f-8e06b63d2e79)
![image](https://github.com/PCBox/PCBox/assets/107446530/9bf98db1-7581-41e3-94ae-0c3768b1ee55)

**Comment by bachimusprime** [comment]
I honestly have no clue what's going on here, the Qt UI code is literally identical to that of 86Box's. It's probably down to the differences in the build scripts and servers 86Box uses for its AppImage builds on their Jenkins buildserver, versus what PCBox uses for its Linux builds on GitHub Actions. 86Box has the same GitHub Actions builds as PCBox though, so this issue would probably occur in those builds as well. Honestly, you should probably just be compiling PCBox yourself if you're using it on Linux. The official 86Box documentation has great guidance on how to do this yourself, which also applies in its entirety to PCBox, except instead of the executable being named `86Box`, it's named `PCBox`.

**Reply by randompersononinternet69** [reply]
What would the point of compiling PCBox make? Am I missing something?
Linux isn't ***all*** about compiling.
I always have issues compiling anyway, mostly because of some dependency issue. If it can't find the dependency, **install it automatically!**

**Comment by randompersononinternet69** [comment, marked as answer]
Sorry, found out I used the wrong executable, turns out I didn't download the Qt one, ended up downloading the non-qt one.
